tests/data/power_flow/asym-line/rmatrix-with-n-xmatrix-with-n-c1-c0/test.dss

```diff
@@ -0,0 +1,26 @@
+Clear
+
+Set DefaultBaseFrequency=50
+
+! Swing or Source Bar
+New circuit.test_1 phases=3 pu=1.0 basekv=10 bus1=Node1.1.2.3.4
+
+! LineCodes
+New LineCode.LineCode1 nphases=4 C1=0.54 C0=0.32 Normamps=115.0 Units=km
+~ Rmatrix = [0.6904 |0.0495 0.6904 |0.0492 0.0495 0.6904 | 0.0495 0.0492 0.0495 0.6904]
+~ Xmatrix = [0.8316 |0.7559 0.8316 |0.7339 0.7559 0.8316 | 0.7559 0.7339 0.7559 0.8316]
```

Comment:
```suggestion
~ Xmatrix = [0.8316 |0.7559 0.8316 |0.7339 0.7559 0.8316 | 0.7559 0.7339 0.7559 0.8316]
~ Neutral=4 Kron=yes ! eliminate the neutral conductor (4)
```
Since we use kron reduction to calculate, better to mention this.